java-11.0/src/Main.java

```diff
@@ -31,11 +33,11 @@ public RuntimeOutput main(RuntimeContext context) throws Exception {
         }
 
         Map<String, Object> json = new HashMap<>();
-        json.put("motto", "Build like a team of hundreds_");
```

Comment:
Let's keep new motto after rebrand. Where did you copy code from? We probably need to replace to new motto there too.